project-base/src/Form/Front/Customer/Password/NewPasswordFormType.php

```diff
@@ -29,7 +30,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'first_options' => [
                     'constraints' => [
                         new Constraints\NotBlank(['message' => 'Please enter password']),
-                        new Constraints\Length(['min' => 6, 'minMessage' => 'Password must be at least {{ limit }} characters long']),
+                        new Constraints\Length(['min' => CustomerUserPasswordFacade::MINIMUM_PASSWORD_LENGTH, 'minMessage' => 'Password must be at least {{ limit }} characters long']),
```

Comment:
The constant is awesome ❤️ . Maybe it could be in entity or password objec instead of facade but it is up to you.

Reply:
If it is up to me, I am okay with current state :)

Reply:
Ok then.